src/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>boardLead</title>
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="#" class="title">boardLead</a>
+    </nav>
+  </header>
+  <main>
+    <section class="parent">
+      <div id="tablediv" class="tablediv">
+        <!-- Table -->
+        <ul>
+          <li class="li1"><strong> 'Продукты' </strong>, <strong>score:</strong> 100</li>
+          <li class="li2"><strong> 'John Banana' </strong>, <strong>score:</strong> 20</li>
+          <li class="li1"><strong> 'Patrick' </strong>, <strong>score:</strong> 10</li>
+          <li class="li2"><strong> 'Carlos' </strong>, <strong>score:</strong> 50</li>
+          <li class="li1"><strong> 'Oscar' </strong>, <strong>score:</strong> 60</li>
+          <li class="li2"><strong> 'Albert Pike' </strong>, <strong>score:</strong> 40</li>
+          <li class="li1"><strong> 'Robert' </strong>, <strong>score:</strong> 70</li>
```

Comment:
- [ ] This content has to be rendered with Javascript using the DOM manipulation, please do so to comply with the project requirements. Don't forget to use ES6 syntax.